indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -872,4 +873,58 @@ private void taskComplete(
     // Notify interested parties
     taskRunnerWorkItem.setResult(taskStatus);
   }
+
+  public Collection<ZkWorker> findLazyworkers(Predicate<ZkWorker> isLazyWorker, int excessWorkers)
```

Comment:
yap, we only want to cleanup excess workers, limiting this outside this will mean that there will be more workers marked as lazy sitting idle without either being assigned any task or being cleaned up by the resource management strategy.